src/main/java/org/graylog/collector/file/FileReaderService.java

```diff
@@ -98,6 +98,11 @@ protected void doStart() {
             return;
         }
 
+        if (paths.isEmpty()) {
+            log.info("No files to follow for input \"{}\" yet, files will be followed once they appear in \"{}\"",
```

Comment:
No because nothing prevents you from creating an input for a file that does not exist yet.